plugins/python-build/bin/python-build

```diff
@@ -166,6 +166,60 @@ can_use_macports() {
   PYTHON_BUILD_SKIP_MACPORTS=1; return 1
 }
 
+can_use_pixi() {
+  if locked_in; then
+    locked_in pixi && rc=$? || rc=$?; return $rc
+  fi
+  [[ -n "$PYTHON_BUILD_USE_PIXI" && -n "$PYTHON_BUILD_SKIP_PIXI" ]] && {
+    echo "error: mutually exclusive environment variables PYTHON_BUILD_USE_PIXI and PYTHON_BUILD_SKIP_PIXI are set" >&3
+    exit 1
+  }
+  [[ -n "$PYTHON_BUILD_USE_PIXI" ]] && return 0
+  [[ -n "$PYTHON_BUILD_SKIP_PIXI" ]] && return 1
+  # Return cached result if already checked
+  [[ -n "$PYTHON_BUILD_PIXI_ENVS_DIR" ]] && return 0
+  # Check if pixi exists and cache the environment directory
+  local envs_dir
+  command -v pixi &>/dev/null && \
```

Comment:
[AFAICS](https://pixi.sh/latest/python/tutorial/#why-is-this-useful), `pixi` can only run on top of an existing `conda` installation. How would it get into PATH?

If the `conda` is going to be installed with Pyenv, there'll be a shim for `pixi` -- so its existence should rather be checked with `pyenv-which pixi` and the conda environment should be active. Otherwise, when checking like this, we'll get `pyenv: pixi: command not found` when trying to run it.